python/lsst/dax/imgserv/cli_v1.py

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env python
```

Comment:
Why does this file exist as a copy of `cli.py`?

Reply:
cli_v1.py is based on v1 whereas cli.py is on v0 of the new JSON schema, hence minor but significant differences.  Eventually cli.py will be retired when we obselete v0 code altogether from imgserv.